apps/src/lib/script-editor/editorRedux.js

```diff
@@ -0,0 +1,278 @@
+import _ from 'lodash';
+
+const INIT = 'scriptEditor/INIT';
+const ADD_GROUP = 'scriptEditor/ADD_GROUP';
+const ADD_STAGE = 'scriptEditor/ADD_STAGE';
+const TOGGLE_EXPAND = 'scriptEditor/TOGGLE_EXPAND';
+const REMOVE_LEVEL = 'scriptEditor/REMOVE_LEVEL';
+const CHOOSE_LEVEL_TYPE = 'scriptEditor/CHOOSE_LEVEL_TYPE';
+const CHOOSE_LEVEL = 'scriptEditor/CHOOSE_LEVEL';
+const ADD_VARIANT = 'scriptEditor/ADD_VARIANT';
```

Comment:
Currently no way to delete a variant once it's added — user has to delete the entire level, and create a new one with the necessary variants.